site/client/CountryProfileConstants.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+    covidCountryProfileSlug,
+    covidLandingSlug,
+    covidCountryProfileRootPath
+} from "site/server/covid/CovidConstants"
+
+export type CountryProfileProject = "coronavirus" | "co2"
+
+export interface CountryProfileSpec {
+    project: CountryProfileProject
+    pageTitle: string
+    genericProfileSlug: string
+    landingPageSlug: string
+    selector: string
+    rootPath: string
+}
+
+export const co2CountryProfileRootPath = "co2/country"
+
+export const countryProfileSpecs: CountryProfileSpec[] = [
```

Comment:
Nit: instead of an array a mapped type might be better (you wouldn't have to do the `finds` later on). Something like:

```typescript
export type CountryProfileProject = "coronavirus" | "co2"

export interface CountryProfileSpec {
    project: CountryProfileProject
    pageTitle: string
}

export const countryProfileSpecs: { [key in CountryProfileProject]: CountryProfileSpec } = {
    coronavirus: {
        project: "coronavirus",
        pageTitle: "Coronavirus Pandemic"
    },
    co2: {
        project: "co2",
        pageTitle: "CO2"
    }
}
```

Reply:
Ah ok, I had played around with using an object, but wanted iterability. ES6 Maps offer both so that's perhaps the better structure.